docs/install/lin/activating-a-validator.md

```diff
@@ -21,6 +21,8 @@ The wallet is scanned for the required amount of Göerli ETH after being linked.
 
 Depending on your platform, issue the appropriate command from the examples below to generate a public / private keypair for your validator.
 
+> NOTICE: When prompted, provide a password to encrypt your new ETH2 validator and withdrawl keys.
```

Comment:
Please ensure you match formatting with these, notice is always bolded (ie ` > **NOTICE:** words`

Reply:
fixed across all files